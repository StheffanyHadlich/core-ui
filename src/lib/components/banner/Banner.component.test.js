import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/banner/__snapshots__"
});
