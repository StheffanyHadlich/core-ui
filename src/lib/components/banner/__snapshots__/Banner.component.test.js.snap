// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Banner Default 1`] = `
<div
  className="sc-ifAKCX krxbjr"
>
  <h3
    className="sc-EHOje ccjuFF"
  >
    Error banner 
  </h3>
  <div
    className="sc-bdVaJa czPhAI sc-banner"
  >
    <i
      className="fas fa-exclamation-triangle"
    />
    <div
      className="sc-bwzfXH kulSxP"
    >
      <div
        className="sc-bxivhb fUzblA"
      >
        Error
      </div>
      <span
        className="sc-htpNat huuJxa"
      >
        There is an error.
      </span>
    </div>
  </div>
  <h3
    className="sc-EHOje ccjuFF"
  >
    Warning banner 
  </h3>
  <div
    className="sc-bdVaJa iTgNUh sc-banner"
  >
    <i
      className="fas fa-exclamation-triangle"
    />
    <div
      className="sc-bwzfXH kulSxP"
    >
      <div
        className="sc-bxivhb fUzblA"
      >
        Warning
      </div>
      <span
        className="sc-htpNat huuJxa"
      >
        There is a warning.
      </span>
    </div>
  </div>
  <h3
    className="sc-EHOje ccjuFF"
  >
    Success banner 
  </h3>
  <div
    className="sc-bdVaJa hXaEkV sc-banner"
  >
    <i
      className="fas fa-check-circle"
    />
    <div
      className="sc-bwzfXH kulSxP"
    >
      <div
        className="sc-bxivhb fUzblA"
      >
        Success
      </div>
      <span
        className="sc-htpNat huuJxa"
      >
        Everything is ok.
      </span>
    </div>
  </div>
</div>
`;
